src/cli/cli_tcp.cpp

```diff
@@ -55,10 +60,33 @@ TcpExample::TcpExample(otInstance *aInstance, OutputImplementer &aOutputImplemen
     , mEndpointConnected(false)
     , mSendBusy(false)
     , mUseCircularSendBuffer(true)
+    , mUseTLS(false)
+    , mTLSHandshakeComplete(false)
     , mBenchmarkBytesTotal(0)
     , mBenchmarkBytesUnsent(0)
 {
+    mEndpointAndCircularSendBuffer.mEndpoint   = &mEndpoint;
+    mEndpointAndCircularSendBuffer.mSendBuffer = &mSendBuffer;
+}
+
+#if OPENTHREAD_CONFIG_TLS_ENABLE
+static int mbedtls_entropy_function(void *ctx, unsigned char *buffer, size_t length)
+{
+    OT_UNUSED_VARIABLE(ctx);
+    otError error = ot::Random::Crypto::FillBuffer(static_cast<uint8_t *>(buffer), static_cast<uint16_t>(length));
+    if (error != OT_ERROR_NONE)
+    {
+        return -1;
+    }
+    return 0;
```

Comment:
nit: prefer a single return statement
```suggestion
    return (error == OT_ERROR_NONE) ? 0 : -1;
```

Reply:
I fixed this in the latest commit.